docs/guides/qiskit-transpiler-service.mdx

```diff
@@ -98,9 +98,9 @@ The following examples demonstrate how to transpile circuits using the Qiskit Tr
 
 Following are the most relevant limitations of the service:
 
-- The Qiskit Transpiler Service can transpile up to 1 million two-qubit gates in either `ai` mode `ai="true"`, `ai="false"`, or `ai="auto"`.
-- The maximum time allowed to run a transpilation process is 30 minutes.
-- You must retrieve the transpilation result from the service within 20 minutes after the transpilation process ends. After 20 minutes, your job is removed.
+- The Qiskit Transpiler Service can transpile up to 1 million two-qubit gates per job in any `ai` mode.
```

Comment:
```suggestion
- The Qiskit Transpiler Service can transpile up to 1 million two-qubit gates per any circuit part of a transpilation job in any `ai` mode.
```

I know the suggestion is not clear wording, yet the point is that it will fail if any of the circuits submitted as part of the same transpilation job to the transpiler service contains > 1 M two-qubit gates

Reply:
Basically a transpilation job could contain N circuits. If any of those N circuits has > 1M two-qubit gates, it will fail

With the current text (IMO), someone could think that the job will fail if it contains >1M two-qubit gates in total (sum of two-qubit gates per circuit or similarly)

Reply:
How about "The maximum number of two-qubit gates _per circuit_ in a transpilation job is 1 million."

Reply:
> > Basically a transpilation job could contain N circuits. If any of those N circuits has > 1M two-qubit gates, it will fail
> > With the current text (IMO), someone could think that the job will fail if it contains >1M two-qubit gates in total (sum of two-qubit gates per circuit or similarly)
> 
> Oh interesting, because I thought it's per job (since that's how Qiskit transpiler works). My apologies.

no worries! Basically my comment was to be more accurate. Thanks for raising the issue @jyu00 :)

Reply:
could we use @abbycross suggestion instead of my own one, @beckykd ?

https://github.com/Qiskit/documentation/pull/2080#discussion_r1787838987

Reply:
@cbjuan - do we want to keep the part about any ai mode?

Reply:
I think so. Thank you!